### PR TITLE
Create StageConfig

### PR DESCRIPTION
**Pull request by imjohsep**
Please merge into your [jwt-config pr](https://github.com/JawboneHealth/cookiecutter-jhhmicroservice/pull/6) branch to satisfy those changes requests

### CODE REVIEW

File: {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/config.py [hunk 1]
```diff
@@ -64,27 +75,23 @@ class DevConfig(BaseConfig):
     AUTO_UPGRADE = True
     {% endif %}
```
Comment (ray-ray) [inline {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/config.py]: I think this {% endif %} should have been deleted, no?

Comment (imjohsep) [reply {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/config.py]: It collapsed the rest of the code. Expand it and you should be able see the `{% if`

Comment (imjohsep) [reply {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/config.py]: It collapsed the rest of the code. Expand it and you should be able see the {% if

Comment (ray-ray) [reply {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/config.py]: Derp. My bad.

**Pull request merged**